tools/OwaspComplianceChecker/create-archive.sh

```
@@ -0,0 +1,548 @@
+echo "Installing dependencies..."
+npm install
+
+# Create necessary directories
+mkdir -p screenshots
+
+echo "Setup complete! You can now run 'npm run dev' to start the application."
+EOL
+chmod +x /tmp/owasp-compliance-checker/scripts/setup.sh
+
+# Create a tar.gz file
+cd /tmp
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)